src/service/impl/departures/gql/jp3/quay-departures.graphql

```diff
@@ -9,11 +15,13 @@ query quayDepartures($id: String!, $numberOfDepartures: Int, $startTime: DateTim
       startTime: $startTime
       timeRange: $timeRange
       includeCancelledTrips: true
+      whiteListed: {lines: $filterByLineIds}
```

Comment:
This is backward compatible so when passing in nothing it works as before? For these files are in use from before, no?

Reply:
Yes, and yes. The query variable is optional, and the default is to not filter anything. You can see a test query [here](https://api.entur.io/graphql-explorer/journey-planner-v3?operationName=quayDepartures&query=query%20quayDepartures%28%0A%20%20%24id%3A%20String%21%2C%0A%20%20%24numberOfDepartures%3A%20Int%2C%0A%20%20%24startTime%3A%20DateTime%2C%0A%20%20%24timeRange%3A%20Int%2C%0A%20%20%24filterByLineIds%3A%20%5BID%5D%0A%29%20%7B%0A%20%20quay%28id%3A%20%24id%29%20%7B%0A%20%20%20%20id%0A%20%20%20%20description%0A%20%20%20%20publicCode%0A%20%20%20%20name%0A%20%20%20%20estimatedCalls%28%0A%20%20%20%20%20%20numberOfDepartures%3A%20%24numberOfDepartures%0A%20%20%20%20%20%20startTime%3A%20%24startTime%0A%20%20%20%20%20%20timeRange%3A%20%24timeRange%0A%20%20%20%20%20%20includeCancelledTrips%3A%20true%0A%20%20%20%20%20%20whiteListed%3A%20%7Blines%3A%20%24filterByLineIds%7D%0A%20%20%20%20%29%20%7B%0A%20%20%20%20%20%20date%0A%20%20%20%20%20%20expectedDepartureTime%0A%20%20%20%20%20%20aimedDepartureTime%0A%20%20%20%20%20%20realtime%0A%20%20%20%20%20%20cancellation%0A%20%20%20%20%20%20quay%20%7B%0A%20%20%20%20%20%20%20%20id%0A%20%20%20%20%20%20%7D%0A%20%20%20%20%20%20destinationDisplay%20%7B%0A%20%20%20%20%20%20%20%20frontText%0A%20%20%20%20%20%20%7D%0A%20%20%20%20%20%20serviceJourney%20%7B%0A%20%20%20%20%20%20%20%20id%0A%20%20%20%20%20%20%20%20line%20%7B%0A%20%20%20%20%20%20%20%20%20%20id%0A%20%20%20%20%20%20%20%20%20%20description%0A%20%20%20%20%20%20%20%20%20%20publicCode%0A%20%20%20%20%20%20%20%20%20%20transportMode%0A%20%20%20%20%20%20%20%20%20%20transportSubmode%0A%20%20%20%20%20%20%20%20%7D%0A%20%20%20%20%20%20%7D%0A%20%20%20%20%7D%0A%20%20%7D%0A%7D%0A&variables=%7B%0A%20%20%22id%22%3A%20%22NSR%3AQuay%3A71184%22%2C%0A%20%20%22numberOfDepartures%22%3A%2010%2C%0A%20%20%22startTime%22%3A%20null%2C%0A%20%20%22timeRange%22%3A%2010000%0A%7D).